### PR TITLE
Generate usable manifest for a project with Dockerfile & Procfile

### PR DESCRIPTION
**Pull request by prognostikos**
## Release Playbook
- [x] Rebase against master
- [ ] Release branch ()
- [ ] Pass CI ()
- [ ] Code review
- [ ] Merge into master
- [ ] Release master ()
- [ ] Pass CI ()
- [ ] Update staging rack
- [ ] Edit [Rack release record](https://github.com/convox/rack/releases) and/or update [docs](https://github.com/convox/convox.github.io)
- [ ] Publish release
- [ ] Release CLI

Before this change, the generated `docker-compose.yml` would not
generate sections for the services defined in the Procfile.

### CODE REVIEW

**Comment by prognostikos** [comment]
So this is kind of a minimal, hackish way of solving the problem I had, but it does work :)

Another option would be to re-work the switch statement in Init()

Open to suggestions - still very much a golang beginner.

**Comment by prognostikos** [comment]
Hm, looking at the other open PRs it seems this will end up conflicting with #591- @ddollar wdyt?

**Comment by ddollar** [comment]
Yes I think it will. #591 attempts to simplify the `convox init` process a bit. Would it be possible to rebase this work on top of that? I'd be happy to jump on a Hangout or something and assist.

**Comment by prognostikos** [comment]
I'll give it a try and ping you if needed.

**Comment by prognostikos** [comment]
This is ready for review.

File: examples/dockerfile-procfile/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM gliderlabs/alpine:3.1
```
Comment (matthewford) [inline examples/dockerfile-procfile/Dockerfile]: how about `ruby:2.3-alpine`?

Comment (prognostikos) [inline examples/dockerfile-procfile/Dockerfile]: @matthewford that's probably a good idea for another PR, and all of the other examples Dockerfiles could be updated as well. AFAIK, images are never built with these Dockerfiles - they're only used when testing various `convox init` scenarios.

**Comment by ddollar** [comment]
I've had some problems with the alpine images and their non-standard libc. How about making these be based on ubuntu:16.04 like convox/rails

**Comment by mattmanning** [comment]
Closing this in favor of https://github.com/convox/rack/pull/683

In that PR I have removed the use of alpine in the test and used our convox/ruby image instead.

**Pull request closed without merging**